op-node/rollup/derive/attributes.go

```diff
@@ -116,14 +116,25 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 		upgradeTxs = append(upgradeTxs, fjord...)
 	}
 
-	l1InfoTx, err := L1InfoDepositBytes(ba.rollupCfg, sysConfig, seqNumber, l1Info, nextL2Time)
+	var hasDeposits = len(depositTxs) > 0
+	l1InfoTx, err := L1InfoDepositBytes(ba.rollupCfg, sysConfig, seqNumber, l1Info, nextL2Time, hasDeposits)
 	if err != nil {
 		return nil, NewCriticalError(fmt.Errorf("failed to create l1InfoTx: %w", err))
 	}
 
-	txs := make([]hexutil.Bytes, 0, 1+len(depositTxs)+len(upgradeTxs))
+	var postDeposits []hexutil.Bytes
+	if ba.rollupCfg.IsInteropActivationBlock(nextL2Time) && hasDeposits {
```

Comment:
I believe `IsInteropActivationBlock` only returns true when its the activation block, aka one particular block, we want the function that always returns true after interop has activated, perhaps `IsInterop`